aperturedb/Connector.py

```diff
@@ -380,8 +380,8 @@ def _query(self, query, blob_array = [], try_resume=True):
                 if self._send_msg(data):
                     response = self._recv_msg()
                     if response is not None:
-                        querRes = queryMessage_pb2.queryMessage()
-                        querRes.ParseFromString(response)
+                        querRes = queryMessage.queryMessage()
+                        querRes.ParseFromString(bytes(response))
```

Comment:
this was added because response is a `bytearray` in 4.x and ParseFromString was expecting a `bytes type object` which `bytearray` *is* but it wouldn't accept.

This may not be the optimal method.

Reply:
Due to this: https://github.com/protocolbuffers/protobuf/issues/10774

Reply:
changed to use memoryview, should be more performant, some people seem to https://stackoverflow.com/questions/55772903/python-memoryview-slower-than-expected say it can be slower than `bytes` but perhaps that is due to it optimizing the copy out in that situation.